src/BookSettings.ts

```diff
@@ -22,18 +22,28 @@ export default class BookSettings {
     private readonly store: Store;
     private readonly bookViews: BookView[];
     private viewLinks: { [key: string]: HTMLAnchorElement };
-    private selectedView: BookView;
+    private readonly fontSizes: string[];
+    private fontSizeLinks: { [key: string]: HTMLAnchorElement };
+
     private viewChangeCallback: () => void;
+    private fontSizeChangeCallback: () => void;
+
+    private selectedView: BookView;
+    private selectedFontSize: string;
+
     private static readonly SELECTED_VIEW_KEY = "settings-selected-view";
+    private static readonly SELECTED_FONT_SIZE_KEY = "settings-selected-font-size";
 
-    public static async create(bookViews: BookView[], store: Store) {
-        const settings = new this(bookViews, store);
+    /** @param fontSizes Array of font size strings sorted from smallest to largest. */
+    public static async create(bookViews: BookView[], fontSizes: string[], store: Store) {
+        const settings = new this(bookViews, fontSizes, store);
         await settings.initializeSelections();
         return settings;
     }
 
-    protected constructor(bookViews: BookView[], store: Store) {
+    protected constructor(bookViews: BookView[], fontSizes: string[], store: Store) {
```

Comment:
It might be a good idea to document that the order of the elements in `fontSizes` will determine the order in which they will be cycled through. It can be reasonably inferred, but still!